bin/webpack-dev-server.js

```diff
@@ -302,12 +315,8 @@ function processOptions(wpOpt) {
 	new Server(compiler, options).listen(options.port, options.host, function(err) {
 		if(err) throw err;
 
-		var uri = url.format({
-			protocol: protocol,
-			hostname: options.host,
-			port: options.port.toString(),
-			pathname: options.inline !== false ? "/" : "webpack-dev-server/"
-		});
+		//add the pathname of uri
+		uri += options.inline !== false ? "/" : "webpack-dev-server/"
```

Comment:
Can you make this a separate variable instead? It's hackish to override the variable in this case since in `query` it's used for something else.